site/_posts/2019-07-29-vote-on-naming-j-release.html.md

```diff
@@ -9,18 +9,18 @@ published: true
+Jakovenko is a Russian chess player. He learned how to play from his father at the age of three years and was later coached by Garry Kasparov's former trainer Alexander Nikitin. In the July 2009 FIDE world rankings Jakovenko became the fifth highest rated chess player in the world and overtook Vladimir Kramnik as the number one Russian.
 - [Jakubowski, Krzysztof](https://en.wikipedia.org/wiki/Krzysztof_Jakubowski) Peak rating 2565 (January 2015)
-Krzysztof Jakubowski is a multiple medalist of the Polish Junior Chess Championship: silver in 1993 (U10), 2002 (U20), bronze in 1995 (U12), 2000 (U18), 2001 (U18). He also won medals in the Polish Junior Team Chess Championship and Polish Junior Rapid Chess Championship.
+Jakubowski is a Polish chess player. He has won multiple medals at the Junior Chess Championships in Poland: silver in 1993 (U10), 2002 (U20), bronze in 1995 (U12), 2000 (U18), 2001 (U18). He also won medals in the Polish Junior Team Chess Championship and Polish Junior Rapid Chess Championship.
 - [Jansa, Vlastimil](https://en.wikipedia.org/wiki/Vlastimil_Jansa) Peak rating 2540 (July 1975)
-Jansa learned chess while in the hospital at the age of eight and at fourteen, became the youth champion of Prague. A deep and original thinker, Jansa has been one of the Czech Republic's leading players for many years, collecting experience across a range of chess activities.
+Jansa is from the Czech Rebublic and learned chess while in the hospital at the age of eight. At fourteen, he became the youth champion of Prague. A deep and original thinker, Jansa has been one of the Czech Republic's leading players for many years, collecting experience across a range of chess activities.
 - [Johannessen, Leif Erlend](https://en.wikipedia.org/wiki/Leif_Erlend_Johannessen) Peak Rating 2564 (October 2005)
 Johannessen is a Norwegian chess player, and Norway's fifth grandmaster. He received his title in 2002, and was at the time Norway's second youngest grandmaster of all time. He picked up his first norm in Oslo, the second at Bermuda and finally the third in the Sigeman tournament in Malmö.
 - [Jones, Gawain](https://en.wikipedia.org/wiki/Gawain_Jones) Peak rating 2702 (May 2019)
-Jones began playing chess at the age of four, competing in his first tournaments at six. In early 1997 he hit the headlines and was featured on the front page of The Guardian newspaper when he became the youngest player in the world ever to beat an International Master in an official tournament game.
+Jones ia British chess player and began playing at the age of four, competing in his first tournaments at six. In early 1997 he hit the headlines and was featured on the front page of The Guardian newspaper when he became the youngest player in the world ever to beat an International Master in an official tournament game.
```

Comment:
ia => is a

Reply:
Ugh, sorry for making you have to mine the text. But appreciate your :eyes:

Reply:
No worries, it's a 'talent' I've been blessed with, my 👀 tend to gravitate towards certain misbehaving texts. Or maybe it's a curse, as sometimes I'm too bothered by the typos/grammar nuances to pay enough attention to the actual content, which is what's important!